server/models/reservation.js

```diff
@@ -9,9 +9,9 @@ module.exports = function (Reservation) {
             }
         }
 
-    Reservation.observe("after save", function (ctx, next) {
+    Reservation.observe("after save", function (ctx, next, asd, dsa, asad) {
         try {
-            Reservation.app.models.Campground.findById(ctx.instance.campgroundId, function (err, campground) {
+            Reservation.app.models.campground.findById(ctx.instance.campgroundId, function (err, campground) {
```

Comment:
I am not receiving an email because the code breaks here.  It does not pick up campground (null).

Reply:
Got it. I'll try to investigate it more soon.